app/shared/src/commonMain/kotlin/ui/subject/episode/EpisodePage.kt

```diff
@@ -132,7 +133,10 @@ private fun EpisodeSceneContent(
     val imageViewer = rememberImageViewerHandler()
     BackHandler(enabled = imageViewer.viewing.value) { imageViewer.clear() }
 
-    ScreenOnEffect()
+    val playbackState = vm.playerState.state.collectAsStateWithLifecycle()
```

Comment:
```suggestion
    val playbackState by vm.playerState.state.collectAsStateWithLifecycle()
```

collectAsState 一般倾向于写 by